normits_demand/models/forecasting/EDGE_growth/loading.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+"""
+Created on: 26/05/2023
+Updated on:
+
+Original author: Ben Taylor
+Last update made by:
+Other updates made by:
+
+File purpose:
+
+"""
+# Built-Ins
+import dataclasses
+import pickle
+from pathlib import Path
+from typing import Union
+
+# Third Party
+import pandas as pd
+
+# Local Imports
+# pylint: disable=import-error,wrong-import-position
+from normits_demand.models.forecasting import forecast_cnfg                                                              
+from normits_demand.utils import file_ops
+from normits_demand.models.forecasting.edge_growth import ticket_splits
+
+
+# pylint: enable=import-error,wrong-import-position
+
+# # # CONSTANTS # # #
+
+# # # CLASSES # # #
+@dataclasses.dataclass
+class TPLoopOutput:
+    overall_not_grown: int
+    overall_base: int
+    factored: dict
+    summary: pd.DataFrame
+    """
+    Return from _tp_loop function in 'run'
+    
+    Parameters
+    ----------
+    overall_not_grown: Sum of matrix movements not grown by the process.
+    overall_base: Sum of base demand.
+    factored: Dictionary of grown matrices. Keys are demand segments.
+    summary: A summary dataframe with rows for each segment.
+    """
+
+@dataclasses.dataclass
+class GlobalVars:
```

Comment:
I'd avoid calling anything global in code - global already has connotations, which can be misleading in this context.

i.e. calling `globals()` in code will return all current global variables